exercises/concept/lasagna/src/test/java/utils/Lasagna.java

```diff
@@ -31,11 +31,11 @@ public int preparationTimeInMinutes(int amountLayers) {
         }
     }
 
-    public int totalTimeInMinutes(int amountLayers, int actualMinutes) {
+    public int totalTimeInMinutes(int amountLayers) {
```

Comment:
This changes the spec from two parameters to one, whereas we _want_ students to use two parameters.

Reply:
I guess I am still not understanding. The goal of this exercise is to have the student use two parameters even though one is never used. I did open a defect, but it was closed as working as expected. But I don't believe that is actually the case since we were working on this one as a group and all of us had the same thought. And we are communicating and this is a good thing. This exercise is blocking our progress in the Java track. And we don't want to submit a solution that is bad practice, because then that looks bad on us if anybody reviews our solution. Please advise

Reply:
@laurahannah Take a look at some other solutions: https://exercism.org/tracks/java/exercises/lasagna/solutions

For example:
```java
public class Lasagna {
    private final int COOKING_TIME_IN_MINS = 40; // cooking time is 40 minutes
    private final int MINS_PREP_PER_LAYER = 2; // each layer takes 2 minutes to cook
    
    // Returns expected length of 40 minutes
    public int expectedMinutesInOven(){
        return COOKING_TIME_IN_MINS;
    }
    // Returns minutes remaining in oven, based on expected minutes
    public int remainingMinutesInOven(int minutesPassed){
        return this.expectedMinutesInOven() - minutesPassed;
    }
    // Returns number of minutes you have spent preparing the lasagna
    public int preparationTimeInMinutes(int numLayers){
        return MINS_PREP_PER_LAYER * numLayers;
    }
    // Returns the total time you have been making the lasagna
    public int totalTimeInMinutes(int numLayers, int minutesPassed){
        return this.preparationTimeInMinutes(numLayers) + minutesPassed;
    }
}
```

You'll see both params are used when the exercise is solved correctly.

Reply:
Can the instructions be made clearer somehow?

Reply:
maybe explain what the second parameter for the totalTimeInMinutes method is. Also, I have a virtually identical solution to one that was accepted 5 months ago that does not pass

Reply:
The instructions say:
>>> Define the totalTimeInMinutes() method that takes two parameters: the first parameter is the number of layers you added to the lasagna, and the second parameter is the number of minutes the lasagna has been in the oven. The function should return how many minutes in total you've worked on cooking the lasagna, which is the sum of the preparation time in minutes, and the time in minutes the lasagna has spent in the oven at the moment.

How could we improve that to make it more clear?

Reply:
I have to agree with the OP.
In my opinion, the method/function name should be `totalElapsedTimeInMinutes`.
When doing this exercise the first couple of times (in different languages) I was confused by this.
In my mind, the method would ask "how many minutes it **should** be in the oven" as apposed to "how many minutes **it has been** in the oven."
I was able to infer the correct behaviour from the tests, but the naming is still confusing

Reply:
I agree 100%. Plus the example in the readme, when added to the unit tests, won't pass. Like, even if you are not talking about method names and such. Add the unit test from the readme, then you have this to solve:
```totalTimeInMinutes(1, 30) = 42
totalTimeIntMintues(3, 20) = 26
totalTimeInMintues(4, 8) = 48```

When in doubt, I went with the README. So, please update the README, which was the first request that I had.

Reply:
> ```
> totalTimeIntMintues(3, 20) = 26
> totalTimeInMintues(4, 8) = 48
> ```

As far as I can tell, the second example isn't in the readme or tests.
The tests have the following test
```java
totalTimeInMinutes(4, 8) == 16
```
which makes total sense given the intended purpose of the method.

Also, the remake's example is 
```java
totalTimeIntMintues(3, 20) == 26
```
which also makes sense given the intended purpose of the method.